suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -959,3 +960,6 @@ export const selectInstacelessUnselectedDevices = memoize((state: DeviceRootStat
 
 export const selectIsBitcoinOnlyDevice = (state: DeviceRootState) =>
     isBitcoinOnlyDevice(selectDevice(state));
+
+export const selectHasBitcoinOnlyFirmware = (state: DeviceRootState) =>
```

Comment:
What's the difference between these two selectors?

Reply:
The first one checks device, the second checks firmware

Reply:
but yes, the name of the selectIsBitcoinOnlyDevice confused me previously as well

Reply:
ddaeae5da7

Reply:
You are right, that is better, Device is confusing whether it is hw device or our suite device. Trezor is much better

fa0dcac981